Resources/views/Form/fields.html.twig

```diff
@@ -204,6 +204,9 @@
         {% if expanded %}
             {% set attr = attr|merge({'class': attr.class|default('') ~ ' ' ~ horizontal_input_wrapper_class}) %}
         {% endif %}
+        {% if layout is sameas(false) %}
```

Comment:
It should be "same as" instead "sameas"

Reply:
Fixed here : https://github.com/phiamo/MopaBootstrapBundle/pull/1215